ext/random/config.m4

```diff
@@ -0,0 +1,20 @@
+dnl
+dnl Check for arc4random on BSD systems
+dnl
+AC_CHECK_DECLS([arc4random_buf])
+
+dnl
+dnl Setup extension
+dnl
+PHP_NEW_EXTENSION(random,
+	  random.c \
+      engine_combinedlcg.c \
+      engine_mt19937.c \
+      engine_pcgoneseq128xslrr64.c \
+      engine_xoshiro256starstar.c \
+      engine_secure.c \
+      engine_user.c \
+      randomizer.c,
+      no,, -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1)
+PHP_INSTALL_HEADERS([ext/random], [php_random.h])
+PHP_ADD_MAKEFILE_FRAGMENT()
```

Comment:
you don't need this here.

Reply:
fix: 32424bab7ca65e07c8a30f2431cff4a3ebffb02c